cvmfs/catalog_mgr_impl.h

```diff
@@ -7,17 +7,13 @@
 
 #define __STDC_FORMAT_MACROS
 
-#include "cvmfs_config.h"
```

Comment:
`cvmfs_config.h` should remain.  This file contains constants created by the build system.